src/Orchard.Web/Modules/Orchard.Taxonomies/Models/TermPart.cs

```diff
@@ -51,6 +51,11 @@ public class TermPart : ContentPart<TermPartRecord> {
             set { Store(x => x.Weight, value); }
         }
 
+        public string FullWeight {
```

Comment:
Because the value is just for querying and  we don't need it when loading a content item, shouldn't we ignore this property in the Infoset? This will prevent to have to keep them both in sync.